File: marioless.c
#include <cs50.h>
#include <stdio.h>

int main(void)

//ask user for height

{
    int size;
    do
    {
        size = get_int("size: ");
    }

    while (size < 1 || size > 8);

//print pyramid

    for (int column = 0 ; column < size; column++)
    {
        for (int space = size - 1 ; space > column; space--)
        {
            printf(" ");
        }
        for (int row = 0; row <= column; row++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario-less.py
def main():
    height = get_height()
    for i in range(height):
        i += 1
        k = height - i
        print(" " * k, end="") #spacing on left side the pyramid
        print("#" * i) #print hashes for pyramid based on user input 


def get_height():
    n = 0
    while n <= 0 or n > 8:
        try:
            n = int(input("Height: "))  # get height from user
            if n > 0 and n <= 8:  # height between 1 and 8
                return n
        except ValueError:  # reprompt for numerical height if user inputs a letter
            print("Input an integer")


main()
